### ISSUE TITLE
Have to make twice to get correct .app file

### ISSUE THREAD

**Issue by jinnipark** [ninenines/erlang.mk#316]
{modules, []} section in compiled .app file is empty on the first build.
The [] part is supposed to include all modules.
It's filled on the second build.

Patched like below to make it work correctly.

```
155a156
 > 		{error, R2} ->
4639,4640c4640
< 	$(eval MODULES := $(shell find ebin -type f -name *.beam | sed "s/ebin\///;s/.beam//;" | tr "\n" "," | sed '$$s/.$$//'))
< #	$(eval MODULES := $(patsubst %,'%',$(sort $(notdir $(basename $(wildcard ebin/*.beam))))))
---
> 	$(eval MODULES := $(patsubst %,'%',$(sort $(notdir $(basename $(wildcard ebin/*.beam))))))
```

**Comment by essen** [comment]
What OS?

-- 
Loïc Hoguin
http://ninenines.eu

-------- Original Message --------

**Comment by jinnipark** [comment]
Mac OS X

**Comment by essen** [comment]
Will check soon, thanks.

**Comment by essen** [comment]
Yep that's me being dumb.

**Comment by essen** [comment]
Fixed in 9dcec593e87f4766cde2ff6486f06de559d3ef65. Thanks!
